Minio/Helper/Utils.cs

```diff
@@ -210,8 +210,7 @@ internal static bool IsSupersetOf(IList<string> l1, IList<string> l2)
 #if NET6_0_OR_GREATER
                 ParallelOptions parallelOptions = new()
                 {
-                    MaxDegreeOfParallelism
-                        = maxNoOfParallelProcesses
+                    MaxDegreeOfParallelism = maxNoOfParallelProcesses, CancellationToken = CancellationToken.None
```

Comment:
This is already the default value, so it doesn't need to be specified (`default` == `CancellationToken.None`). A better solution would be to pass the `CancellationToken` to `ForEachAsync` and use it here too.

Reply:
Changed the code.
Please let me know if the change addresses your comment.